src/crimson/os/seastore/transaction_manager.h

```diff
@@ -316,6 +316,20 @@ class TransactionManager : public ExtentCallbackInterface {
     });
   }
 
+  template <typename T>
+  read_extent_ret<T> get_mutable_extent_by_laddr(Transaction &t, laddr_t laddr, extent_len_t len) {
+    return read_extent<T>(t, laddr, len
```

Comment:
follow up https://github.com/ceph/ceph/pull/50419/commits/d8a114fd738043b182185553f1f7f73e136b332a#r1375737128

How about something like:
```c++
return get_pin(t, offset
).si_then([this, &t](auto pin) {
  ceph_assert(!pin->is_zero_reserved());
  ceph_assert(!pin->is_clone());
  return this->read_pin<T>(t, std::move(pin));
}).si_then(
...
```
?

Reply:
Sorry, I didn't catch the comment. Done.